mkdocs.yml

```diff
@@ -430,7 +430,7 @@ pages:
         - shipctl utility reference: platform/tutorial/workflow/using-shipctl.md
         - runSh Usecases: platform/tutorial/workflow/using-runSh.md
         - Run job directly on node: platform/tutorial/workflow/run-job-on-node.md
-        - How to create Jira issues for CI and runSh jobs: platform/tutorial/workflow/create-jira-issues.md
+        - How to create Jira issues for CI and pipelines jobs: platform/tutorial/workflow/create-jira-issues.md
```

Comment:
- pipelines -> "assembly line"
- The article title should match https://github.com/Shippable/docs/pull/1182/files#diff-f0f11412847bc054ad1d70fe9dfde9dfR19